cli/users.go

```diff
@@ -140,6 +141,42 @@ var cmdUsers = []cobra.Command{
 				return
 			}
 
+			logOK()
+		},
+	},
+	{
+		Use:   "enable <user_id> <user_auth_token>",
+		Short: "Change user status to active",
```

Comment:
`Change user status to enabled` ?

Reply:
Please leave "avtive"

Users cannot be enabled, they are not devices. Users are active/inactive and/or deactivated. Please keep this terminology, not enabled/disabled.

Reply:
Why a user can't be disabled? For example, Azure is using this therminology here: https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-mfa-userstates#azure-ad-multi-factor-authentication-user-states

Inactive can refer to a user not using the platform for a long time, right? In any case I agree to chose a naming and to be consistent in all the code.